andela/trime
# End to End tests for local authentication.

#### What does this PR do?
Adds end to end local authentication tests.

#### Description of Task to be completed?
Write e2e tests using Protractor and Selenium to test the local authentication workflow.

#### How should this be manually tested?
Clone this repo, install the dependencies by running
```
npm install
```
Make sure you have Java (JDK) installed in your system then run:
```
npm run e2esetup
```
This will spin up the Selenium webserver and install Selenium weddriver if it doesn't exist in your system.
Then run:
```
gulp e2etest
```

#### What are the relevant pivotal tracker stories?
[E2E Test Configuration](https://www.pivotaltracker.com/projects/1454736/stories/117129339)
[E2E Test: For authentication](https://www.pivotaltracker.com/projects/1454736/stories/117129333)

@hisabimbola @andela-ekupara @andela-kndungu
